handlers/playback.go

```diff
@@ -36,14 +37,32 @@ func PlaybackHandler() httprouter.Handle {
 		}
 		defer response.Body.Close()
 
+		w.Header().Set("accept-ranges", "bytes")
```

Comment:
Should this be set on OPTIONS response as well?

Reply:
I don't think it's needed as it's only to check supported http methods right? but wouldn't do any harm i guess